src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -188,14 +189,46 @@ public void SetAccessControl_FileStream_FileSecurity_Success()
         }
 
         [Fact]
-        public void Create_Directory_From_DirectoryInfo()
+        public void DirectoryInfo_Create_NullDirectorySecurity()
+        {
+            DirectoryInfo info = new DirectoryInfo("path");
+            Assert.Throws<ArgumentNullException>(() => info.Create(null));
+        }
+
+
+        [Fact]
+        public void DirectoryInfo_Create_DefaultDirectorySecurity()
+        {
+            DirectorySecurity security = new DirectorySecurity();
+            CreateAndVerifyDirectoryWithSecurity(security);
+        }
+        [Theory]
+        [InlineData("S-1-5-32-544", FileSystemRights.FullControl, AccessControlType.Allow)]
```

Comment:
I would not use `Administrators` as you will potentially block tests when run by accounts that are not in the group. Use [`WellKnownSidType`] to be clearer, and give the rights to `BuiltinUsersSid`, which should be the `Users` group. You can validate what is what by getting/setting permissions in the UI and getting/setting the ACL with the current extension methods.

Reply:
Ok I'll use that.

Reply:
Done.